user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Style_guide.md

```diff
@@ -0,0 +1,281 @@
+
+   |☀️  Light background |🌙 Dark background|
+   |---------------------|----------------|
+   |#F4F4F5              |#1E2530         |
+
+- Make sure the background color of your component style is different from the page/panel background. This way, components can easily be distinguished from the background. This can be used for query filters, tables, timelines, etc. (anything but titles, images, and buttons). Generally, it is best to choose a slightly lighter color than that of the page/panel background. Here are some good default colors:
+
+   |☀️  Light background |🌙 Dark background|
+   |------------------|----------------|
+   |#FDFDFD           |#272E38        |
+   |![Component light style](~/user-guide/images/LCA_Style_Guide_Component_Style_Light.png)|![Component dark style](~/user-guide/images/LCA_Style_Guide_Component_Style_Dark.png)|
+
+- When you pick colors for pages, panels, buttons, grids, and tables, there **must** be enough contrast between text and icons and their background. Otherwise, users will have trouble seeing the text and icons.
+
+   |👍 Good |👎 Bad |
+   |------|-----|
+   |![Good contrast](~/user-guide/images/LCA_Style_Guide_Contrast_Good.png)|![Bad contrast](~/user-guide/images/LCA_Style_Guide_Contrast_Bad.png)|
+
+- When you pick the main color for a low-code app, also make sure there is enough contrast. When you select an app color, the app itself will determine what the color of the text is, based on the selected background color. Be sure to pick a color that results in enough contrast between the text and background color, especially in the pages menu on the left, as this will be a slightly darker color than the selected app color.
+
+   |👍 Good |👎 Bad |
+   |------|-----|
+   |![Good app color](~/user-guide/images/LCA_Style_Guide_App_Color_Good.png)|![Bad app color](~/user-guide/images/LCA_Style_Guide_App_Color_Bad.png)|
+
+- The colors for states in tables and charts **must** be used consistently, so that the same color is always used to represent the same state.
+
+   ![State color](~/user-guide/images/LCA_Style_Guide_Color_State.png)
+
+- Limit the number of colors that you use in a dashboard or app. Avoid making it too colorful or too monotone.
+
+- Picking good colors to start from is often a challenge. One way to approach this is to pick some colors from a layout you find online, like this sample below:
+
+   |Sample UI | Color palette | Resulting app with same color palette|
+   |----------|---------------|-----------------------|
+   |![Sample UI](~/user-guide/images/LCA_Style_Guide_Sample_UI.webp)|![Color palette](~/user-guide/images/LCA_Style_Guide_Color_Palette.png)|![Sample UI result](~/user-guide/images/LCA_Style_Guide_Sample_UI_Result.png)|
+
+## Layout
+
+- Make sure components on a page/panel are aligned vertically and horizontally.
```

Comment:
On this example, wouldn't we use the same margin for the Contacts table as we have between the pie chart and the column chart?